cypress/integration/dashboardPremium.js

```diff
@@ -4,7 +4,9 @@ describe('Dashboards Premium Features', () => {
         cy.location('pathname').should('include', '/dashboard')
     })
 
-    it('Tag dashboard', () => {
+    // Taggables are an enterprise feature. Cypress isn't setup with a scale license so these
+    // tests should fail now that we make that license check in the backend and return a 402.
+    xit('Tag dashboard', () => {
```

Comment:
Still wondering if there's anything to do with that. Probably worth a new issue to follow up, if no such issue exists?

Reply:
Agreed, I'll open up an issue for this

Reply:
https://github.com/PostHog/posthog/issues/8654